package.json

```diff
@@ -31,6 +31,7 @@
     "after": "0.8.1",
     "istanbul": "0.3.9",
     "mocha": "2.2.5",
+    "replacestream": "~4.0.0",
```

Comment:
It looks like replacestream is not installable on the default Node.js 0.8 npm. Can you pick a different dependency for the testing, please? Also, please only specify exact versions of dependencies outside of pillarjs/jshttp.